src/Test/Plutip/Internal/Cluster.hs

```diff
@@ -896,7 +901,10 @@ generateGenesis
     -> IO GenesisFiles
 generateGenesis dir systemStart initialFunds addPoolsToGenesis extraConf = do
     source <- getShelleyTestDataPath
-    Yaml.decodeFileThrow @_ @Aeson.Value (source </> "alonzo-genesis.yaml")
+    let alonzoGenesis = if (ecIncreasedExUnits extraConf)
+                        then "alonzo-genesis-increased-exUnits.yaml"
+                        else "alonzo-genesis.yaml"
+    Yaml.decodeFileThrow @_ @Aeson.Value (source </> alonzoGenesis)
```

Comment:
Could we not directly modify the `Aeson.Value`, before encoding it into the json file?

Reply:
In reference to https://github.com/mlabs-haskell/plutip/pull/165#discussion_r1050334139

Reply:
I like it. See [cdf6623](https://github.com/mlabs-haskell/plutip/pull/165/commits/cdf6623a11947a991d7e322ac6eb7e9959874987)